[2022-06-28 14:17:00,571] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-28T14:16:00+00:00 [queued]>
[2022-06-28 14:17:00,579] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-28T14:16:00+00:00 [queued]>
[2022-06-28 14:17:00,579] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:17:00,579] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-28 14:17:00,579] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-28 14:17:00,586] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): get_datetime> on 2022-06-28 14:16:00+00:00
[2022-06-28 14:17:00,591] {standard_task_runner.py:52} INFO - Started process 106 to run task
[2022-06-28 14:17:00,593] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2022-06-28T14:16:00+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/firstdemo.py', '--cfg-path', '/tmp/tmpvhbki0ci', '--error-file', '/tmp/tmpmdde7ux6']
[2022-06-28 14:17:00,593] {standard_task_runner.py:80} INFO - Job 218: Subtask get_datetime
[2022-06-28 14:17:00,629] {logging_mixin.py:109} INFO - Running <TaskInstance: first_***_dag.get_datetime scheduled__2022-06-28T14:16:00+00:00 [running]> on host ca68cb5b3870
[2022-06-28 14:17:00,666] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-06-28T14:16:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-28T14:16:00+00:00
[2022-06-28 14:17:00,667] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-28 14:17:00,668] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-06-28 14:17:00,678] {subprocess.py:85} INFO - Output:
[2022-06-28 14:17:00,680] {subprocess.py:89} INFO - Tue Jun 28 14:17:00 UTC 2022
[2022-06-28 14:17:00,681] {subprocess.py:93} INFO - Command exited with return code 0
[2022-06-28 14:17:00,696] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220628T141600, start_date=20220628T141700, end_date=20220628T141700
[2022-06-28 14:17:00,724] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-28 14:17:00,744] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
